packages/fetch-router/demos/cf-workers/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "lib": ["ES2024", "DOM", "DOM.Iterable"],
+    "module": "ES2022",
+    "target": "ESNext",
+    "moduleResolution": "Bundler",
+    "resolveJsonModule": true,
+    "allowJs": true,
+    "checkJs": false,
+    "noEmit": true,
+    "isolatedModules": true,
+    "allowSyntheticDefaultImports": true,
+    "forceConsistentCasingInFileNames": true,
+    "allowImportingTsExtensions": true,
+    "rewriteRelativeImportExtensions": true,
+    "verbatimModuleSyntax": true,
+    "skipLibCheck": true,
+    "types": ["./worker-configuration.d.ts", "node"]
+  },
+  "include": ["worker-configuration.d.ts", "worker.ts", "../../global.d.ts"]
```

Comment:
Would love some input on how to get rid of the `../../global.d.ts` inclusion here

I could put [a copy](https://github.com/remix-run/remix/blob/main/packages/fetch-router/global.d.ts) in each demo, but that feels kind of weird as well to just keep propagating this file